clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -993,7 +993,8 @@ readSYCLImagesFromTable(StringRef TableFile, const ArgList &Args) {
 ///
 /// \returns A path to the LLVM Module that contains wrapped images.
 Expected<StringRef> wrapSYCLBinariesFromFile(StringRef InputFile,
-                                             const ArgList &Args) {
+                                             const ArgList &Args,
+                                             bool IsEmbeddedIR = false) {
```

Comment:
Do we need a default value here? Is there any other users of this function outside of `runWrapperAndCompile` that already provides a default?

Reply:
Good point. We do not need the default here.